ebl-issuance/src/main/java/org/dcsa/conformance/standards/eblissuance/party/EblIssuanceCarrier.java

```diff
@@ -190,6 +197,40 @@ private void sendIssuanceRequest(JsonNode actionPrompt) {
             .formatted(isCorrect ? "correct" : "incorrect", tdr, eblStatesByTdr.get(tdr)));
   }
 
+  private ObjectNode getSupportingDocumentObject(){
+    var document = generateDocument();
+    return OBJECT_MAPPER.createObjectNode()
+    .put("name","test-iss-document")
+      .put("content",document)
+      .put("mediatype","application/octet-stream");
+  }
+
+  @SneakyThrows
+  private static byte[] generateDocument() {
+    var uuid = UUID.randomUUID();
+    byte[] pdf;
+    String filepath = "/standards/eblissuance/messages/test-iss-document.pdf";
+    try (InputStream inputStream = EblIssuanceCarrier.class.getResourceAsStream(filepath)) {
+      if (inputStream == null) {
+        throw new IllegalArgumentException("Could not resolve " + filepath);
+      }
+      pdf = inputStream.readAllBytes();
+    }
+    var docWithUuid = new byte[pdf.length + 16];
+
+    // Add a UUID to ensure it is unique, such that the testers do not have to "remove" the
+    // document between every test.
+    System.arraycopy(pdf, 0, docWithUuid, 0, pdf.length);
+    putLong(uuid.getMostSignificantBits(), docWithUuid, pdf.length);
+    putLong(uuid.getLeastSignificantBits(), docWithUuid, pdf.length + 8);
+    return docWithUuid;
```

Comment:
I do not know the PDF format well enough here. Can we just blindly throw a binary UUID at the end of the document without corrupting it?